stats/Record.md

```diff
@@ -0,0 +1,73 @@
+# Record API Overview
+The stats library allows users to record metrics for their applications or third-party libraries.
+ The core data types used are:
+* `Measure`: describes the type of the individual values recorded by an application.
+* `Measurement`: describes a data point to be collected for a `Measure`.
+
+## Measure
+A `Measure` describes the type of the individual values/measurements recorded by an application. It
+includes information such as the type of measurement, the units of measurement and descriptive names
+for the data. This provides the fundamental type used for recording data.
+
+A Measure describes a value with the following metadata:
+* `name`: a string by which the measure will be referred to, e.g. "rpc_server_latency", or
```

Comment:
Same as View names. Measure names should be ASCII string with no more than 255 characters.

Reply:
We can add later a section with restrictions for all types.